specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5773,13 +5773,238 @@
         },
         "configuration": {
           "type": "object",
+          "properties": {
+            "parsingMode": {
+              "type": "string",
+              "required": false,
+              "$ref": "#/definitions/ParsingMode"
```

Comment:
Seems weird to have both a `type` and a `$ref` in the same property. I think just the `$ref` should be sufficient.

Reply:
Yes, the type is also defined in the $ref so this is redundant.  Will remove all instances.

Reply:
Fixed in commit 18b17f8ce55b9ca0c51bb93c7ed7a26c567fa8ed.